homeassistant/components/obihai/manifest.json

```diff
@@ -2,6 +2,6 @@
   "domain": "obihai",
   "name": "Obihai",
   "documentation": "https://www.home-assistant.io/integrations/obihai",
-  "requirements": ["pyobihai==1.2.3"],
+  "requirements": ["pyobihai==1.3.0"],
```

Comment:
In addition to changing the version here you also need to perform the second bullet point: https://developers.home-assistant.io/docs/development_checklist

Reply:
Thanks!

Reply:
I tried updating the pyobihai version in requirements_all.txt manually, and there was still a failed check.  I ran the gen_requirements_all script and uploaded the resulting requirements_all.txt, but that had many conflicts, so I changed it back.  I don't have a proper development environment for HA and github.  I've just made been coding on my "production" server (my raspberry pi hassio setup) and uploading the files to github.com.

Reply:
The remaining failed check (pylint exits with code 4) seems to be unrelated to requirements_all.txt.  Does it just not like the comment?